extensions/blocks/story/story.php

```diff
@@ -168,19 +168,33 @@ function render_video( $media ) {
 	}
 
 	$metadata = wp_get_attachment_metadata( $media['id'] );
+
 	if ( ! empty( $metadata ) && ! empty( $metadata['videopress'] ) ) {
+		// Use poster image for VideoPress videos.
 		$poster_url  = $metadata['videopress']['poster'];
 		$description = ! empty( $metadata['videopress']['description'] ) ? $metadata['videopress']['description'] : '';
+		$meta_width  = $metadata['videopress']['width'];
+		$meta_height = $metadata['videopress']['height'];
```

Comment:
Do you think we'd need to add some defaults here in case those dimensions are not set?

Reply:
Probably a good idea to just leave out the width/height attribute if it's not set, as we were doing before. I did that in 6b984c3.

Reply:
I think I would still have a empty string when it's not set, to avoid PHP notices:

```suggestion
		$meta_width  = ! empty( $metadata['videopress']['width'] ) ? $metadata['videopress']['width'] ) : '';
		$meta_height = ! empty( $metadata['videopress']['height'] ) ? $metadata['videopress']['height'] : '';
```